### PR TITLE
[FEAT] Grub Shop

### PR DESCRIPTION
**Pull request by 0xsunflowerstudios**
# Description

Add UI functionality + event to fulfill an order at the Grub Shop 🚀 

Screen Shot 2022-10-07 at 3 54 11 pm
Screen Shot 2022-10-07 at 3 54 18 pm

### CODE REVIEW

File: src/features/game/types/consumables.ts [hunk 1]
```diff
@@ -26,4 +32,166 @@ export const CONSUMABLES: Record<ConsumableName, Consumable> = {
       Egg: new Decimal(1),
     },
   },
+  "Bumpkin Broth": {
+    name: "Bumpkin Broth",
+    description: "Boiled Eggs are always a good breakfast choice",
```
Comment (mGranTY) [inline src/features/game/types/consumables.ts:37]: repeated details for the new consumables.

File: src/features/game/types/images.ts [hunk 2]
```diff
@@ -1057,6 +1057,14 @@ export const ITEM_DETAILS: Items = {
     image: egg,
     description: "A boiled egg",
   },
+  "Bumpkin Broth": {
+    image: greenEgg,
+    description: "A boiled egg",
```
Comment (mGranTY) [inline src/features/game/types/images.ts:1062]: same with the description here.

File: src/features/game/lib/constants.ts [hunk 3]
```diff
@@ -524,6 +528,53 @@ export const INITIAL_FARM: GameState = {
     ],
   },
   bumpkin: INITIAL_BUMPKIN,
+
+  grubShop: {
+    opensAt: new Date("2022-10-05").getTime(),
+    closesAt: new Date("2022-10-08").getTime(),
+    orders: [
+      {
+        id: "asdj123",
```
Comment (mGranTY) [inline src/features/game/lib/constants.ts:537]: is these ids right?

**Comment by adamhannigan** [comment]
@mGranTY Thanks for the review! Yes all of this data is currently placeholders, we are missing real images + descriptions

**Review by adamhannigan — APPROVED** [review]
Another great improvement from Sunflower Studios!

**Pull request merged**